slang-gfx.h

```diff
@@ -1339,6 +1339,7 @@ struct GraphicsPipelineStateDesc
     DepthStencilDesc    depthStencil;
     RasterizerDesc      rasterizer;
     BlendDesc           blend;
+    bool                meshShader;
```

Comment:
Why is this needed? We can tell from `program` by checking whether or not it includes a mesh shader stage. It's best to avoid this change because it increases the chance of bugs due to forgetting to set this bit and issues regarding binary compatibility.

Reply:
It wasn't immediately clear how to extract this information from `ShaderProgramBase`, I've done it now in [`1c00cab` (#3190)](https://github.com/shader-slang/slang/pull/3190/commits/1c00cab1fea3472c788e19b3e98c8bd3dc118861)